### ISSUE TITLE
Validation set results

### ISSUE THREAD

**Issue by tianyu-su** [ekazakos/temporal-binding-network#6]
Hello! Thank you for the code and the pretrained models.  However, I loaded the "epic_tbn_rgbflowaudio.pth.tar" from the pretrained directory, the terminal showed "model epoch 68 best prec@1: 25.38076362129323". So I am confused that is it the best checkpoint which is trained on the whole training set? If not, can you publish the best checkpoint to verify the results in README last section?

**Comment by ekazakos** [comment]
Hi,

This is indeed the best checkpoint of the model trained on the whole training set. **25.38** is the best **action** accuracy on both S1 and S2, which was computed by taking the geometric average of the action accuracies of S1 and S2. 

Best,

Vangelis

**Comment by tianyu-su** [comment]
OK, thank you for your answer.
